packages/bridge/src/runtime/nitro/renderer.ts

```diff
@@ -148,7 +150,8 @@ export default defineRenderHandler(async (event) => {
     error: ssrError,
     redirected: undefined,
     nuxt: undefined as undefined | Record<string, any>, /* Nuxt 2 payload */
```

Comment:
In nuxt 2, nuxt could not have hooks.
(It is probably a payload, so it needs to be serializable.)